R/series.R

```diff
@@ -416,7 +415,7 @@ reeval_dots <- function(x, series.short, verbose = TRUE) {
 
     # additional options that are required to produce a series
     requires.i <- as.character(SPECS[SPECS$short == series.NA.i & SPECS$is.series, ]$requires)
-    if (!identical(requires.i, "")){
+    if (!(identical(requires.i, "") || is.na(requires.i))) {
```

Comment:
Could we save an empty `requires` as `""` so we don't need this?

Reply:
I used `readr::write_csv` to store the csv which put `NA` into the file instead of empty string so this grew dynamically.

Ultimately we can encode this however we want.
In my mind `NA` conveyed "does not need additional arguments" better than `""` but thinking on it a little more this could also be construed as "we don't know'. Maybe `NULL`? `""` somehow seems like the least desirable option to me.

Reply:
I would prefer not to change the existing code if not really needed. Can we write the same `""` as before to `SPECS.csv`?